src/lint-changed.ts

```diff
@@ -35,12 +35,12 @@ async function runCommand(cmdString: string) {
   if (stderr) {
     throw new Error(stderr);
   }
-  return stdout;
+  return stdout.trim();
```

Comment:
This was ultimately the issue. The `gitLastTag` method returned the tag name padded with a newline. That actually broke the diffing mechanism to find file changes.

Reply:
😄